Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "kudo"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+prost = "0.10.3"
+tokio = { version = "1.18.1", features = ["full"] }
```

Comment:
Do we need `full` features from tokio?

Reply:
Indeed we don't. Fixed it in latest commit.